### DISCUSSION TITLE
远程桌面连接失败

### DISCUSSION THREAD

**Discussion by onlyALL0** [natfrp/wiki#152] [使用问题] [answered]
请教下各位：
隧道列表设置了本地ip或者127.0.0.1，端口号3389，也开了自动https。访问认证可以通过，但是认证通过后再次访问隧道链接时，提示**ERR_EMPTY_RESPONSE**。
这时候用远程桌面去访问时，提示无法连接到该计算机。
ps：1、远程桌面各种设置都做了，内网直接通过ip地址就可以远程。
       2、外网在openwrt做了端口转发，用公网ip+端口号也可以远程。
       3、防火墙设置过了，开启或者关闭都无法通过隧道地址连接成功。

**Comment by onlyALL0** [comment]
有没有设置成功的，请帮我看看有没有地方不对劲

**Reply by tidymoon** [reply]
印象之中部分节点是不能穿透RDP流量的，但是我找不到那篇公告了，建议更换节点尝试（？

**Reply by onlyALL0** [reply]
多谢，关闭https后成功了

**Comment by fakeboboliu** [comment, marked as answer]
请关闭 自动https

我们大概需要考虑把这个也塞到高级用户模式里面去

**Reply by onlyALL0** [reply]
感谢，关闭自动https后成功了
